analyzers/src/SonarAnalyzer.Common/Common/AnalyzerLanguage.cs

```diff
@@ -130,6 +130,22 @@ public string ResourceSuffix
             }
         }
 
+        public string HelpLink(string id)
```

Comment:
This whole class is very ugly. I'm not going to fix it here, but I already have other prerequisite PRs in progress to fix this.

Reply:
Fixed in #5662 - should be merged first, so we'll have a rebase conflict here.